src/app/course/course-item/course-item.component.spec.ts

```diff
@@ -1,25 +1,39 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
 
-import { CourseItemComponent } from './course-item.component';
+import { CourseItemComponent } from "./course-item.component";
+import { ActionTypes } from "../_models/actionTypes";
```

Comment:
talking about the models, there is no need to name directory with underscore.
If models are related to one component it is fine to put them on the same as component level.
If models are used across the app, you can create app/core/models directory.
If models are related to any service, it is better to put them on service level
app/core/services/my-service/models

Also, could you please use import from barrels? like ../models/index
It looks much better to import a lot of objects from one place.

Fore example, to import services which a are on app/core/services/...
You will just need in component:
`import { MyFirstService, MyFirstServiceModels, MySecondService, MyThirdService } from "../core/index";`